### ISSUE TITLE
Not quiet when uploading thumbnail

### ISSUE THREAD

**Issue by Mister-42** [porjo/youtubeuploader#194]
**Command**:
```bash
${HOME}/bin/youtubeuploader -quiet -cache ${HOME}/.config/youtubeuploader/name.token \
    -filename /tmp/epic-202409.mp4 \
    -thumbnail /tmp/epic-202409.png \
    -metaJSON /tmp/epic-202409.json \
    -sendFilename=false \
    -notify=false > /dev/null
```

**Expected output**:
```bash

```
Nothing indeed, because of ` > /dev/null`

**Real output**:
```bash
2024/10/02 10:34:56 Uploading thumbnail "/tmp/epic-202409.png"...
```

**Suspected cause**:
https://github.com/porjo/youtubeuploader/blob/bc34ad7cb023f7856cdf4017feb7fc5869a1dc38/run.go#L144
```diff
-		log.Printf("Uploading thumbnail %q...\n", config.Thumbnail)
+		fmt.Printf("Uploading thumbnail %q...\n", config.Thumbnail)

```
---
I actually expected the program to be completely quiet when the `-quiet` switch is used without the need to redirect to `/dev/null`, except when errors occur. But that's a different issue perhaps.

**Comment by Mister-42** [comment]
I actually expected the program to be completely quiet when the `-quiet` switch is used, except when errors occur. But that's a different issue perhaps.

**Comment by porjo** [comment]
As per flag usage:
```
 -quiet
    	suppress progress indicator
```
It just stops the progress indicator. Other messages are still displayed. I guess we could add a `-silent` flag...but I'm not sure that's necessary - there are only ever a handful of messages output.

**Comment by Mister-42** [comment]
Fair description about `-quiet`, but main issue has not been adressed.

```
${HOME}/bin/youtubeuploader -quiet -cache ${HOME}/.config/youtubeuploader/name.token \
	-filename /tmp/epic-202409.mp4 \
	-thumbnail /tmp/epic-202409.jpg \
	-metaJSON /tmp/epic-202409.json \
	-metaJSONout ${archive}/${name}/${filename}.json \
	-sendFilename=false \
	-notify=${2} 1> ${HOME}/capture.txt 2> ${HOME}/error.txt
```

Captured _stdout_ to _capture.txt_:
```bash
% cat capture.txt 
Uploading file "/tmp/epic-202410.mp4"

Upload successful! Video ID: [removed]
Wrote video metadata to file "/path/to/epic-202410.json"
Thumbnail uploaded!
Video added to playlist "removed" (PLREMOVED)
```

**This is not correct**. This message should have been sent to _stdout_ instead of _stderr_:
```bash
% cat error.txt 
2024/10/04 09:33:05 Uploading thumbnail "/tmp/epic-202410.jpg"...
```

**Comment by porjo** [comment]
ok, I get it now. I've pushed a fix in [24.02](https://github.com/porjo/youtubeuploader/releases/tag/24.02)
